Project.toml

```diff
@@ -4,10 +4,14 @@ authors = ["JinraeKim <kjl950403@gmail.com> and contributors"]
 version = "0.10.0"
 
 [deps]
+DifferentialEquations = "0c46a032-eb83-5123-abaf-570d42b7fbaa"
```

Comment:
I think it's only for test; do we have to add these dependencies?

Reply:
Ah, okay. 6894d5a considers these dependencies as test-specific dependencies.